### PR TITLE
@orta => Bidder Details Notification

### LINKED ISSUE #357: Add "Send me my details" to Help Shelf
I think this will be the "path to least resistance" to get people their credentials 

access the help button, tap "Send me my Details" enter email and send off details to email. If email is not found, surface appropriate dialogue 

image

image

image

image

### PR DESCRIPTION
**Pull request by ashfurrow**
Fixes #357. Obviously still a work in progress :)

@katarinabatina Between the user entering their email and us showing a confirmation screen, there is going to be a delay (we need to get a confirmation from the API about whether or not this user is even registered). What should we show in the interim? Might last anywhere from 100ms to several seconds, depending on network connection speed. Maybe something like [this](https://github.com/TransitApp/SVProgressHUD)?

Next steps:

- [x] Remove call to old, removed API
- [x] Call new API
- [x] Figure out interim UI
- [x] Retry ability

### CODE REVIEW

**Comment by ashfurrow** [comment]
Heard back from Katarina about the design – due to the limitations of the API, and the fact that it's a pretty minor issue, we're just going to go with a spinner for a minimum of 1s between hitting "enter" and getting "ok, sent!"/"user not found".

File: Kiosk/App/AppDelegate+GlobalActions.swift [hunk 1]
```diff
@@ -128,6 +120,12 @@ private extension AppDelegate {
             self.hideHelpSignal()
         }
     }
+    
+    func showBidderDetailsRetrievealSignal() -> RACSignal {
```
Comment (orta) [inline Kiosk/App/AppDelegate+GlobalActions.swift]: retrieval

File: Kiosk/App/BidderDetailsRetrieval.swift [hunk 2]
```diff
@@ -1,35 +1,77 @@
...
-extension UIViewController {
-    func promptForBidderDetailsRetrieval() {
-        let alertController = UIAlertController.emailPromptAlertViewController() { (email: String) -> () in
-            self.receivedEmail(email)
-        }
-        presentViewController(alertController, animated: true, completion: nil)
+public extension UIViewController {
+    func promptForBidderDetailsRetrievalSignal() -> RACSignal {
+        return RACSignal.createSignal { (subscriber) -> RACDisposable! in
+            let (alertControler, command) = UIAlertController.emailPromptAlertController()
```
Comment (orta) [inline Kiosk/App/BidderDetailsRetrieval.swift]: `alertControler`

**Comment by orta** [comment]
Rockin. All looks good :👍

**Comment by ashfurrow** [comment]
Coooool thanks!

**Pull request merged**